recipes/flask-sspi/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "flask-sspi" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-SSPI-{{ version }}.tar.gz
+  sha256: 23440e3b950cb4ab3cf4f5b8079848b9ee1eb6d8d95a2ee34a4d659237439b48
+
+build:
+  skip: True  # [not win or py<36]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

Actually it can't as it can't be build on Linux, sorry about the mix up 😄

---

Nominally the fact that this is only makes sense for Windows isn't a problem for `noarch: python` as the Windows-only `pywin32` dependency would prevent it from being installed on Windows.

For conda-forge however, the CI only knows how to build `noarch` packages for Linux (as that's much more common) so this recipe is correct.

---

Gotcha, thanks for the clarification.